src/hooks/useSwap.ts

```diff
@@ -48,7 +50,8 @@ const MESSAGE_MAP = {
 };
 
 export const useSwap = ({ swapIn, swapOut, variant, userId, max, chainId }: UseSwapProps) => {
-  const paraSwap = getParaswap(chainId);
+  const { currentChainId } = useProtocolDataContext();
```

Comment:
hmm, an easier(imho more readable) fix would have been to just take `currentNetworkConfig.baseChainId || currentNetworkConfig.chainId`

Reply:
Updated it to use underlyingChainId, as its the one that user sets up when adding fork